Wikipedia/Code/ExploreViewController.swift

```diff
@@ -12,17 +12,34 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
 
     // MARK - UIViewController
     
+    func setAutomaticTheme(background: String) {
+        let userInfo: [String: Any] = [ReadingThemesControlsViewController.WMFUserDidSelectThemeNotificationThemeNameKey: background]
+        NotificationCenter.default.post(name: Notification.Name(ReadingThemesControlsViewController.WMFUserDidSelectThemeNotification), object: nil, userInfo: userInfo)
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         layoutManager.register(ExploreCardCollectionViewCell.self, forCellWithReuseIdentifier: ExploreCardCollectionViewCell.identifier, addPlaceholder: true)
+       
+        let now = Date()
+        let six_today = now.dateAt(hours: 6, minutes: 0)
+        let seventeen_today = now.dateAt(hours: 17, minutes: 00)
 
+        if now >= six_today &&
+          now <= seventeen_today
+        {
+            setAutomaticTheme(background: "light")
+        } else {
+            setAutomaticTheme(background: "dark")
+        }
+        
         navigationItem.titleView = titleView
         navigationBar.addUnderNavigationBarView(searchBarContainerView)
         navigationBar.isUnderBarViewHidingEnabled = true
         navigationBar.displayType = .centeredLargeTitle
         navigationBar.shouldTransformUnderBarViewWithBar = true
         navigationBar.isShadowHidingEnabled = true
-
+        print("viewDidLoad")
```

Comment:
We also remove print statements that have been used to get visibility over our code before opening PRs.